src/Recruiter/Worker.php

```diff
@@ -60,6 +73,8 @@ public function work()
             );
             return (string) $job->id();
         } else {
+            //TODO:! check memory limit??
```

Comment:
@gabrielelana we decided to skip this check for now. We want to avoid a useless save on the database and the memory check will be done at the end of the next executed job. This should not allocate memory if no jobs are executed.